src/_includes/partials/header.njk

```diff
@@ -1,40 +1,41 @@
 {% set otherLang = "fr" if locale == "en" else "en" %}
-<header>
-    <div  lang="{{ otherLang }}">
+<header class="header">
+    <div class="header__container" lang="{{ otherLang }}">
+        <a href="{% if locale == "en" %}/en/ {% else %}/fr/ {% endif %}" class="header__logo">
+        {% if locale == "en" %}
+            {{ '/src/images/en/gov-canada-logo.svg' | svgContents | safe }}
+        {% else %}
+            {{ '/src/images/fr/gov-canada-logo.svg' | svgContents | safe }}
+        </a>
+        {% endif %}
+
         {# loop though site.languages #}
         {% for lgg in site.languages %}
-        {% if loop.first %}<ul>{% endif %}
-
-        {# set translatedUrl to the homepage of that language by default #}
-        {% set translatedUrl = "/" + lgg.code + "/" %}
-
-        {# Only render opposite language link #}
-        {% if lgg.code != locale %}
-
-        {# loop through all the content of the site #}
-        {% for item in collections.all %}
-
-            {# for each item in the loop, check if
-            - its translationKey matches the current item translationKey
-            - its locale matches the code of the language we are looping through #}
-            {% if item.data.translationKey == translationKey and item.data.locale == lgg.code %}
-                {% set translatedUrl = item.url %}
+            {% if loop.first %}<ul>{% endif %}
+
+            {# set translatedUrl to the homepage of that language by default #}
+            {% set translatedUrl = "/" + lgg.code + "/" %}
+
+            {# Only render opposite language link #}
+            {% if lgg.code != locale %}
+                {# loop through all the content of the site #}
+                {% for item in collections.all %}
+                    {# for each item in the loop, check if
+                    - its translationKey matches the current item translationKey
+                    - its locale matches the code of the language we are looping through #}
+                    {% if item.data.translationKey == translationKey and item.data.locale == lgg.code %}
+                        {% set translatedUrl = item.url %}
+                    {% endif %}
+                {% endfor %}
+
+                <li>
+                    <a href="{{ translatedUrl | url }}">{{ lgg.label }}</a>
+                </li>
             {% endif %}
 
-        {% endfor %}
-
-        <li>
-            <a href="{{ translatedUrl | url }}">{{ lgg.label }}</a>
-        </li>
-
-        {% endif %}
-
-        {% if loop.last %}</ul>{% endif %}
+            {% if loop.last %}</ul>{% endif %}
         {% endfor %}
     </div>
 
     {% include "partials/nav.njk" %}
-
-    {% include "partials/breadcrumbs.njk" %}
```

Comment:
I think we should keep the breadcrumbs in the header  element under the main nav.

Reply:
Done!